Homework1/UnitTest1.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MailRuClassLibrary;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Homework1
+{
+    public class Tests
+    {
+        const string Username = "udodudodovich@mail.ru";
+        const string Password = "cxzASDewq123567";
+        const string Url = "https://mail.ru/";
+
+        By MailAddressSelector = By.XPath("//span[@class='ph-project__user-name svelte-1hiqrvn']");
+
+        [Test]
+        public void Test1()
+        {
+            WebDriver driver = new ChromeDriver { Url = Url };
+            driver.Manage().Window.Maximize();
+
+            MailRuHomePage homePage = new MailRuHomePage(driver);
+            homePage.Login(Username, Password);
+
+            var mailAddress = driver.FindElement(MailAddressSelector);
+            var value = mailAddress.Text;
+            Assert.AreEqual(Username, value);
```

Comment:
Don't forget to add the third parameter for assert method. It will allow to receive more understandable reason why test fails if it happens at this line. For example, "Username has wrong value after login"

Reply:
Added the third parameter into AreEqual() method showing text message if the test fails.